src/form/components/StaticField.tsx

```diff
@@ -37,31 +45,50 @@ export type Props = {
 } & (
   | {
       subtype: StaticFieldType.Button
+      title?: string
       button: ButtonProps
     }
   | {
       subtype: StaticFieldType.Text
+      title?: string
       text: string
     }
   | {
       subtype: StaticFieldType.Checklist
+      title?: string
       checklist: Array<Checklist>
     }
 )
 
 export const StaticField = (
-  props: Props & { fieldIndex: number; localeNamespace?: LocaleNamespace },
+  props: Props & { fieldIndex: number; localeNamespace?: LocaleNamespace; formReadOnly: boolean },
 ) => {
   const theme = useFormTheme()
+
   const getCSSStyle = useCSSStyles(theme, 'staticField')({})
   const getInlineStyle = useInlineStyle(theme, 'staticField')({})
   const getRowStyle = useCSSStyles(theme, 'row')({})
 
+  const appTheme = useAppTheme()
+  const getLabelStyle = useAppCSSSTyles(appTheme, 'label')({})
+
   const label = { localeNamespace: props.localeNamespace, ...props.label }
-  return (
+
+  return props.formReadOnly ? null : (
     <FieldRowWrapper key={`row-${props.fieldIndex}`} {...getRowStyle('wrapper')}>
-      {props.label && <Label {...label} />}
+      {props.label ? (
+        <Label {...label} />
+      ) : (
+        <LabelWrapper {...getLabelStyle('wrapper')}>
+          <br />
+        </LabelWrapper>
+      )}
+
       <StaticWrapper {...getCSSStyle('wrapper')}>
+        {props.title && (
```

Comment:
Why do we need a title? Seems like either the field has a label, or it is within a FormFieldGroup that has a title

Reply:
See comment above. It is just adapting the existing styles from before your refactoring. Title might be a little confusing. Maybe it's more description, listTitle... But it is not a formField label and not a group or section title. but heading for the list which again is aligned differently to the rest.

Reply:
wouldn't this be a checklist title then?